### ISSUE TITLE
Jekyll 3.9 container unavailable

### ISSUE THREAD

**Issue by smacz42** [envygeeks/jekyll-docker#310]
The 3.9 series has been released since 2020, with the latest 3.9.1 having just been released recently: https://jekyllrb.com/news/2021/04/08/jekyll-3-9-1-released/. Is there anything plans in the works to get these versions pushed up?

**Comment by juanuys** [comment]
Yes, please - it would be nice to test agaisnt the version officially supported by Github: https://pages.github.com/versions/

**Comment by juanuys** [comment]
Seems a PR exists at #293

**Comment by DanubeRS** [comment]
Gemfile update to 3.9 using the existing 3.8 image (and a bundle update) works OK from what I can tell.

**Comment by mmm8955405** [comment]
3.9.1 when can I use it

**Comment by tavlima** [comment]
Hey, @envygeeks, do you mind clarifying your closing comment in #293? It would really be great to develop locally against 3.9.2, which is [the version currently used by GitHub Pages](https://pages.github.com/versions/)...

**Comment by envygeeks** [comment]
There is nothing to clarify, I misinterpreted that PR, I've re-opened it and will see if the author can fix conflicts and get it out

**Comment by envygeeks** [comment]
The old Jekyll 3.9 PR path is no longer the direction for this repository. Current master has a dedicated jekyll/jekyll:pages target for the GitHub Pages stack, so I am closing this as superseded by that refreshed image path.
